lib/all_your_base_cases.rb

```diff
@@ -1,4 +1,8 @@
 class AllYourBaseCase < OpenStruct
+  def self.build(row:, index:)
+    new(PreProcessor.new(row: row, index: index).run)
```

Comment:
You can get rid of this method completely by just calling the preprocessor directly on line 89